src/lashup_kv.erl

```diff
@@ -406,3 +474,23 @@ handle_full_update(_Payload = #{key := Key, vclock := RemoteVClock, map := Remot
 
 increment_lclock(N) ->
   N + 1.
+
+-spec(matchspec_kv2(ets:match_spec()) -> ets:match_spec()).
+matchspec_kv2({{Key}, Conditions, [true]}) ->
+  {#kv2{key = Key, map = '_',
+        vclock = '_', lclock = '_'},
+   Conditions, ['$_']};
```

Comment:
I would align it a little better. Maybe
`{#kv2{},
   Conditions,
    ['$_']}

Reply:
That's a matchspec, it requires explicitly mark all fields as `'_'`. In your example only `{kv2, undefined, undefined, undefined, undefined}` will be matched.

Reply:
I see. Done.